firstvoices/backend/views/image_views.py

```diff
@@ -81,7 +81,7 @@ def get_queryset(self):
         site = self.get_validated_site()
         return (
             Image.objects.filter(site__slug=site[0].slug)
-            .prefetch_related("site", *get_select_related_media_fields(None))
+            .select_related(*get_select_related_media_fields(None))
```

Comment:
If I recall, Nolan switched this because although it added additional queries it reduced the overall response time. Could you verify that this is an improvement in response time as well as the query count?

Reply:
Yes, I'm not sure about other views, but I have verified that there is no time difference in using select or prefetch related in these couple of viewsets.